runtime/oti/j9nonbuilder.h

```diff
@@ -5516,6 +5516,7 @@ typedef uintptr_t ContinuationState;
 #define J9VM_CONTINUATION_RETURN_FROM_SYNC_METHOD_JNI 5
 
 #define J9VM_CONTINUATION_RUNTIMEFLAG_JVMTI_CONTENDED_MONITOR_ENTER_RECORDED 0x1
+#define J9VM_CONTINUATION_RUNTIMEFLAG_PENDING_DECOMPILE_ALL_METHODS 0x2
```

Comment:
This confused me for a while - perhaps change it from imperative, e.g. `J9VM_CONTINUATION_RUNTIMEFLAG_ALL_METHODS_DECOMPILED`